### PR TITLE
implement cache auto refresh

### LINKED ISSUE #2: auto-refresh cache data
todo 

- [x] check update date
- [x] ask the user whether "refresh now" or "do it later"

### PR DESCRIPTION
**Pull request by hirosassa**
fix #2 

# What is this PR

This PR added cache refresh functionality that check if the cache file is new enough and refresh if it is expired and the user confirmed.

### CODE REVIEW

**Comment by hirosassa** [comment]
@snowhork @to-lz1 Please review!

File: cmd/root.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ var config Config
 type Config struct {
 	BigqueryProjects []string
 	CacheFile        string
+	CacheRefresh     int
```
Comment (snowhork) [inline cmd/root.go]: I guess `CacheRefreshHour` is better.

Comment (hirosassa) [reply cmd/root.go]: Thanks! fixed!

**Comment by snowhork** [comment]
@hirosassa LGTM! thanks!

**Pull request merged**